src/context/orderbook.tsx

```diff
@@ -133,8 +135,25 @@ export const OrderBookProvider: FC<{ children: ReactNode }> = ({ children }) =>
       pairName: activeProduct.pairName,
       devnet: isDevnet
     })
-    const orderbookBids = res.data?.bids.map((item) => [item.price, item.size])
-    const orderbookAsks = res.data?.asks.map((item) => [item.price, item.size])
+    const lastUpdated = new Date(res.data?.updatedTime).getTime()
```

Comment:
All the code below that work with timestamp doesn't make sense if data is nullable.